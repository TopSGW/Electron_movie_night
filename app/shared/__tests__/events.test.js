'use strict'

// This snapshot test serves as a reminder to update all usages of
// event names whenever any name is updated.
const events = require('../events')

/* globals test, expect */
test('event names have not changed', () => {
  expect(events).toMatchSnapshot()
})
