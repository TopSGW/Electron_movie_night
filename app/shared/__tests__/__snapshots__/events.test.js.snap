// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`event names have not changed 1`] = `
Object {
  "CRAWL_COMPLETE": "crawl-complete",
  "CRAWL_DIRECTORY": "crawl-directory",
  "CRAWL_START": "crawl-start",
  "DELETE_MOVIE_DATABASE": "delete-movie-database",
  "LOAD_MOVIE_DATABASE": "load-movie-database",
  "LOG_MESSAGE": "log-message",
  "MOVE_MOVIE_TO_TRASH": "move-movie-to-trash",
  "MOVIE_DATABASE": "movie-database",
  "SEARCHING_DIRECTORY": "searching-directory",
  "SELECT_IMPORT_DIRECTORY": "select-import-directory",
  "UPDATE_MOVIE_METADATA": "update-movie-metadata",
}
`;
